packages/flutter_tools/test/integration.shard/test_test.dart

```diff
@@ -5,79 +5,84 @@
 import 'dart:async';
 import 'dart:convert';
 
-import 'package:flutter_tools/src/artifacts.dart';
 import 'package:flutter_tools/src/base/file_system.dart';
 import 'package:flutter_tools/src/base/io.dart';
-import 'package:flutter_tools/src/cache.dart';
-import 'package:flutter_tools/src/globals.dart' as globals;
 
-import '../../src/common.dart';
-import '../../src/context.dart';
+import '../src/common.dart';
```

Comment:
For the second part of this change this test needs to be changed to be more selective about running pub